packages/bundler/src/BundlerCollectorTracer.ts

```diff
@@ -21,9 +21,9 @@ declare function isPrecompiled (addr: any): boolean
  */
 export interface BundlerCollectorReturn {
   /**
-   * storage and opcode info, collected between "NUMBER" opcode calls (which is used as our "level marker")
+   * storage and opcode info, collected on top-level calls from EntryPoint
    */
-  numberLevels: NumberLevelInfo[]
+  topLevelCalls: TopLevelCallInfo[]
```

Comment:
Is it clear what "top level" means here? Maybe we need some better term for these?
Or just `callsFromEntryPoint`?

Reply:
I tried to keep the parser generic, but I guess we really can't...
I guess you're right, and it will make it clearer.